integrationtests/tests/fixtures/mod.rs

```diff
@@ -97,7 +97,7 @@ pub async fn fixtures(
         tracing_subscriber::fmt()
             .with_env_filter(
                 EnvFilter::try_from_default_env()
-                    .unwrap_or_else(|_| EnvFilter::new("info,tide=error,minimint::consensus=warn")),
+                    .unwrap_or_else(|_| EnvFilter::new("info,axum=error,minimint::consensus=warn")),
```

Comment:
not sure if this is ok

Reply:
You can remove this `axum=error`, too. This just removes `info` and `warning` log levels from output during testing. I presume tide was kind of noisy. But I just ran the `lightning_gateway_pays_invoice` integration test without the `axum=error` and there wasn't any noisy log output so i think we can just remove it.